examples/replica/main.tf

```diff
@@ -0,0 +1,39 @@
+##############################################################################
+# Resource Group
+##############################################################################
+
+module "resource_group" {
+  source = "git::https://github.com/terraform-ibm-modules/terraform-ibm-resource-group.git?ref=v1.0.5"
+  # if an existing resource group is not set (null) create a new one using prefix
+  resource_group_name          = var.resource_group == null ? "${var.prefix}-resource-group" : null
+  existing_resource_group_name = var.resource_group
+}
+
+##############################################################################
+# ICD postgresql database
+##############################################################################
+
+module "postgresql_db" {
+  source            = "../.."
+  resource_group_id = module.resource_group.resource_group_id
+  name              = "${var.prefix}-primary"
+  region            = var.region
+  resource_tags     = var.resource_tags
+}
+
+##############################################################################
+# ICD postgresql read-only-replica
+##############################################################################
+
+module "replicate_postgresql_db" {
+  source            = "../.."
+  resource_group_id = module.resource_group.resource_group_id
+  name              = "${var.prefix}-read-only-replica"
+  region            = var.region
+  resource_tags     = var.resource_tags
+  remote_leader_crn = var.postgresql_db_remote_leader_crn == null ? module.postgresql_db.crn : var.postgresql_db_remote_leader_crn
```

Comment:
This logic is unnecessary in an example. The example always creates a primary, so it can always use it. This makes var.postgresql_db_remote_leader_crn redundant.

Reply:
Removed the variable `postgresql_db_remote_leader_crn`.